.github/workflows/ci.yaml

```diff
@@ -0,0 +1,112 @@
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  lint-python:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - run: |
+          pip install "$(grep -E '^flake8' django/requirements-test.txt)"
+          pip install "$(grep -E '^mypy' django/requirements-test.txt)"
```

Comment:
2 lines necessary because the mypy line has spaces in it; could pipe the grep through sed to put extra quotation marks around it but this is simpler.